frameworks/kafka/tests/test_check_topic_list_on_zk_restart.py

```diff
@@ -38,15 +42,14 @@ def zookeeper_server(configure_security):
         sdk_install.install(
             config.ZOOKEEPER_PACKAGE_NAME,
             config.ZOOKEEPER_SERVICE_NAME,
-            config.ZOOKEEPER_TASK_COUNT,
+            ZOOKEEPER_TASK_COUNT,
             package_version=config.ZOOKEEPER_PACKAGE_VERSION,
-            additional_options=service_options,
+            additional_options=custom_options,
```

Comment:
Shouldn't this be service_options? as there is no custom_options defined outside the if scope, plus inside if statement the dictionaries will be merged to service_options?